integration-tests/tests/specs/features/file-sharing.spec.ts

```diff
@@ -31,6 +31,14 @@ test('submit a single sequence with two files', async ({ pageWithGroup, page })
     const reviewPage = await submissionPage.submitSequence();
     await reviewPage.waitForZeroProcessing();
 
+    // check that files can be seen after processing
+    const filesDialog = await reviewPage.viewFiles();
+    expect(filesDialog.getByText('hello.txt')).toBeVisible();
```

Comment:
I'm not saying this will fix your issue - but we normally `await` `expects` I think